### PR TITLE
Remove POC Zoe fee purse, expiry, etc

### PR DESCRIPTION
**Pull request by erights**
Stacked on #4222  but probably best reviewed directly against master.

### CODE REVIEW

File: packages/treasury/src/stablecoinMachine.js [hunk 1]
```diff
@@ -175,13 +174,7 @@ export const start = async (zcf, privateArgs) => {
...
     };
 
-    return zcf.makeInvitation(
-      makeLoanHook,
-      'MakeLoan',
-      undefined,
-      HIGH_FEE,
-      LONG_EXP,
-    );
+    return zcf.makeInvitation(makeLoanHook, 'MakeLoan', undefined);
```
Comment (Chris-Hibbert) [inline packages/treasury/src/stablecoinMachine.js]: The undefined in final position is no longer necessary.  It's an optional param.

File: packages/treasury/test/faucet.js [hunk 2]
```diff
@@ -25,13 +24,7 @@ export async function start(zcf, privateArgs) {
...
     }
 
-    return zcf.makeInvitation(
-      faucetHook,
-      'provide RUN',
-      undefined,
-      LOW_FEE,
-      SHORT_EXP,
-    );
+    return zcf.makeInvitation(faucetHook, 'provide RUN', undefined);
```
Comment (Chris-Hibbert) [inline packages/treasury/test/faucet.js]:
```suggestion
    return zcf.makeInvitation(faucetHook, 'provide RUN);
```

File: packages/vats/src/bootstrap.js [hunk 3]
```diff
@@ -170,7 +148,7 @@ export function buildRootObject(vatPowers, vatParameters) {
           chainTimerService,
           nameAdmins,
           priceAuthority,
-          zoeWPurse: zoeWUnlimitedPurse,
+          zoeWPurse: zoe,
```
Comment (Chris-Hibbert) [inline packages/vats/src/bootstrap.js]: Would you mind changing `install-on-chain.js` to not use the name zoeWPurse?

File: packages/zoe/src/zoeService/zoe.js [hunk 4]
```diff
@@ -59,98 +50,22 @@ const makeZoeKit = (
...
-  const installFeeAmount = AmountMath.make(feeBrand, zoeFeesConfig.installFee);
-  const startInstanceFeeAmount = AmountMath.make(
-    feeBrand,
-    zoeFeesConfig.startInstanceFee,
-  );
-  const offerFeeAmount = AmountMath.make(feeBrand, zoeFeesConfig.offerFee);
-
-  const { makeFeePurse, chargeZoeFee, feeCollectionPurse } = setupMakeFeePurse(
-    feeIssuer,
+  const { feeMintAccess, getFeeIssuerKit, feeIssuer, feeBrand } = createFeeMint(
```
Comment (Chris-Hibbert) [inline packages/zoe/src/zoeService/zoe.js:59]: It looks like `createFeeMint()` still creates `initialFeeFunds`. Would you remove that from `feeMint.js`?   And drop `feeIssuerConfig.initialFunds`, I think.

File: packages/zoe/src/contracts/vpool-xyk-amm/swap.js [hunk 5]
```diff
@@ -40,22 +39,10 @@ export const makeMakeSwapInvitation = (zcf, provideVPool) => {
...
 
   const makeSwapInInvitation = () =>
-    zcf.makeInvitation(
-      swapIn,
-      'autoswap swapIn',
-      undefined,
-      LOW_FEE,
-      SHORT_EXP,
-    );
+    zcf.makeInvitation(swapIn, 'autoswap swapIn', undefined);
```
Comment (Chris-Hibbert) [inline packages/zoe/src/contracts/vpool-xyk-amm/swap.js]:
```suggestion
    zcf.makeInvitation(swapIn, 'autoswap swapIn);
```

File: packages/zoe/src/contracts/vpool-xyk-amm/swap.js [hunk 6]
```diff
@@ -40,22 +39,10 @@ export const makeMakeSwapInvitation = (zcf, provideVPool) => {
...
 
   const makeSwapOutInvitation = () =>
-    zcf.makeInvitation(
-      swapOut,
-      'autoswap swapOut',
-      undefined,
-      LOW_FEE,
-      SHORT_EXP,
-    );
+    zcf.makeInvitation(swapOut, 'autoswap swapOut', undefined);
```
Comment (Chris-Hibbert) [inline packages/zoe/src/contracts/vpool-xyk-amm/swap.js]:
```suggestion
    zcf.makeInvitation(swapOut, 'autoswap swapOut);
```

File: packages/zoe/src/zoeService/types.js [hunk 7]
```diff
@@ -39,8 +39,12 @@
  * Return an object with the instance, installation, description, invitation
  * handle, and any custom properties specific to the contract.
  * @property {GetFeeIssuer} getFeeIssuer
- * @property {MakeFeePurse} makeFeePurse
- * @property {BindDefaultFeePurse} bindDefaultFeePurse
+ * @property {() => Promise<Purse>} makeFeePurse
+ * Deprecated. Does nothing useful but provided during transition so less old
+ * code breaks.
+ * @property {(defaultFeePurse: ERef<Purse>) => ZoeService} bindDefaultFeePurse
```
Comment (Chris-Hibbert) [inline packages/zoe/src/zoeService/types.js:45]: Isn't it gone now? Can't this be dropped?  What still refers to it?

File: packages/zoe/test/unitTests/zcf/test-zcfSeat-exit.js [hunk 8]
```diff
@@ -45,7 +42,7 @@ test(`zoe - wrongly throw zcfSeat.exit()`, async t => {
   });
 
   // eslint-disable-next-line no-unused-vars
-  const { creatorFacet } = await E(zoe).startInstance(
+  const { _creatorFacet } = await E(zoe).startInstance(
```
Comment (Chris-Hibbert) [inline packages/zoe/test/unitTests/zcf/test-zcfSeat-exit.js]: can the preceding comment ("no-unused-vars") be dropped?

**Review by Chris-Hibbert — CHANGES_REQUESTED** [review]

File: packages/treasury/src/stablecoinMachine.js (hunk 1 above)
Comment (erights) [reply packages/treasury/src/stablecoinMachine.js]: Done, thanks

File: packages/treasury/test/faucet.js (hunk 2 above)
Comment (erights) [reply packages/treasury/test/faucet.js]: Done

File: packages/vats/src/bootstrap.js (hunk 3 above)
Comment (erights) [reply packages/vats/src/bootstrap.js]: Done

File: packages/zoe/src/zoeService/zoe.js (hunk 4 above)
Comment (erights) [reply packages/zoe/src/zoeService/zoe.js:59]: Good catch! Done

File: packages/zoe/src/contracts/vpool-xyk-amm/swap.js (hunk 5 above)
Comment (erights) [reply packages/zoe/src/contracts/vpool-xyk-amm/swap.js]: Done

File: packages/zoe/src/contracts/vpool-xyk-amm/swap.js (hunk 6 above)
Comment (erights) [reply packages/zoe/src/contracts/vpool-xyk-amm/swap.js]: Done

File: packages/zoe/src/zoeService/types.js (hunk 7 above)
Comment (erights) [reply packages/zoe/src/zoeService/types.js:45]: Nothing in agoric-sdk, but other dapps still do, so I'm leaving in this placeholder as a transitional measure. Resolving without change.

(I actually started out removing it, but then CI slapped be with dapp failures. As it should!)

File: packages/zoe/test/unitTests/zcf/test-zcfSeat-exit.js (hunk 8 above)
Comment (erights) [reply packages/zoe/test/unitTests/zcf/test-zcfSeat-exit.js]: Done

**Review by Chris-Hibbert — APPROVED** [review]
Looks Good!  thanks.

**Pull request merged**